kakao/src/main/kotlin/com/agoda/kakao/Actions.kt

```diff
@@ -439,6 +440,35 @@ interface DrawerActions : BaseActions {
 }
 
 /**
+ * Provides action for RatingBar
+ */
+interface RatingBarActions : BaseActions {
+    /**
+     * Set Rating for Rating Bar
+     * @param number of rating to set for the RatingBar
+     */
+    fun setRatingAt(number: Float) {
+        view.perform(object : ViewAction {
+            override fun getDescription() = "performing RatingBarAction: $number"
+
+            override fun getConstraints() = Matchers.allOf(ViewMatchers.isAssignableFrom(View::class.java),
```

Comment:
You don't have to put this kind of syntax here. You can set correct constraints just doing this:
```
override fun getConstraints() = ViewMatchers.isAssignableFrom(RatingBar::class.java)
```